KeepingYouAwake.xcodeproj/project.pbxproj

```diff
@@ -1019,10 +1019,11 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = KeepingYouAwake/KeepingYouAwake.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_STYLE = Manual;
```

Comment:
Can you please exclude any changes of this file from the pull request? These are only required on your local machine.